includes/class-newspack-popups-segmentation.php

```diff
@@ -181,5 +184,36 @@ function( $cat ) {
 	public static function get_segmentation_endpoint() {
 		return plugins_url( '../api/segmentation/index.php', __FILE__ );
 	}
+
+	/**
+	 * Create the clients and events tables.
+	 */
+	public static function create_database_table() {
+		global $wpdb;
+		$events_table_name = Segmentation::get_events_table_name();
+
+		// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching
+		if ( $wpdb->get_var( $wpdb->prepare( 'SHOW TABLES LIKE %s', $events_table_name ) ) != $events_table_name ) {
+			$charset_collate = $wpdb->get_charset_collate();
+
+			$sql = "CREATE TABLE $events_table_name (
+				id bigint(20) NOT NULL AUTO_INCREMENT,
+				created_at date NOT NULL,
+				-- type of event
+				type varchar(20) NOT NULL,
+				-- Unique id of a device/browser pair
+				client_id varchar(100) NOT NULL,
+				-- Article ID
+				post_id bigint(20),
+				-- Article categories IDs
+				categories_ids varchar(100),
```

Comment:
Suggest changing column name to `category_ids` or `categories`.

Reply:
Done in 27d76d966b45a1806853eb1d5a98872d9678eb98